src/kete/rust/flux/common.rs

```diff
@@ -84,15 +84,14 @@ pub fn solar_flux_py(dist: f64, wavelength: f64) -> PyResult<f64> {
 /// emissivity :
 ///     Emissivity of the object, 0.9 by default.
 #[pyfunction]
-#[pyo3(name = "sub_solar_temperature", signature = (obj2sun, geom_albedo, g_param, beaming, emissivity=None))]
+#[pyo3(name = "sub_solar_temperature", signature = (obj2sun, geom_albedo, g_param, beaming, emissivity=0.9))]
```

Comment:
The version bump of pyo3 which introduced this macro requirement actually also removed the need for the Option<f64>, I just didnt notice this at the time.
These changes below do functionally nothing except remove lines and remove the Option, as the default values are in the signature above.